pom.xml

```diff
@@ -2809,7 +2809,7 @@
         <hive.version>${hive23.version}</hive.version>
         <hive.version.short>2.3.4</hive.version.short>
         <hive.parquet.group>org.apache.parquet</hive.parquet.group>
-        <hive.parquet.version>1.8.1</hive.parquet.version>
+        <hive.parquet.version>${parquet.version}</hive.parquet.version>
```

Comment:
then we can remove the `hive.parquet.version` property?

Reply:
Good suggestion.